components/VerticalTabs/MemberList.tsx

```diff
@@ -0,0 +1,33 @@
+// import { useInView } from "react-intersection-observer";
+import MemberCard, { MemberCardProps } from "../MemberCard";
+
+interface MemberListProps {
+    data: MemberCardProps[]
+}
+
+const MemberList = ({data}: MemberListProps) => {
+    return (
+        <div className="flex flex-col">
+            <div className="flex flex-row p-4">
+                <h1 className="text-green-300 text-4xl pl-4">80+ </h1>
+                <h1 className="text-white pt-2 px-4 text-xl"> Members</h1>
+                <h1 className="text-green-300 pl-6 text-4xl">800+ </h1>
+                <h1 className="text-white pt-2 px-4 text-xl"> Alumni</h1>
```

Comment:
Let's use a predefined color for the numbers, change `text-green-300` to `text-wato-teal`